ProcessLib/Process.h

```diff
@@ -235,8 +235,31 @@ class Process
     }
 
 protected:
+    /** This function is for general cases, in which all equations of the
+     coupled processes have the same number of unknowns. For the general cases
+     with the staggered scheme, all equations of the coupled processes share one
+     DOF table hold by \verb{_local_to_global_index_map}. Other cases can be
+     considered by overloading this member function in the
+     derived class.
+     */
     virtual void constructDofTable();
 
+    /**
+     * Construct the DOF table for a specified process in the monolithic scheme,
+     * which is stored in the
+     * member of this class, \verb{_local_to_global_index_map}.
+     */
+    void constructDofTableOfSpecifiedProsessMonolithicScheme();
+
+    /**
+     * Construct the DOF table for a specified process in the staggered scheme,
+     * which is stored in the
+     * member of this class, \verb{_local_to_global_index_map}.
+     * @param specified_prosess_id The specified process ID.
```

Comment:
No need to repeat variable's name in the comment. Drop.

Reply:
Dropped.